### PR TITLE
Add El Pais Translator

### PR DESCRIPTION
**Pull request by adam3smith**
supersedes #1146

### CODE REVIEW

File: El Pais.js [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+	
+	translator.setHandler('itemDone', function (obj, item) {
+		//improve author parsing
+		item.creators = [];
+		var authors = attr(doc, 'meta[name=author]', 'content');
+		authors = authors.split(/\s*,\s/);
+		//Z.debug(authors)
+		for (let author of authors) {
+			if (author !== "Agencias") {
+				item.creators.push(ZU.cleanAuthor(author, "author"))
```
Comment (zuphilip) [inline El Pais.js]: Add `;` at the end.

File: El Pais.js [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://elpais.com/tag/estados_unidos/a/",
+		"items": "multiple"
+	},
```
Comment (zuphilip) [inline El Pais.js]: Add another test case for search page, e.g. https://elpais.com/buscador/?qt=zotero

File: El Pais.js [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+		authors = authors.split(/\s*,\s/);
+		//Z.debug(authors)
+		for (let author of authors) {
+			if (author !== "Agencias") {
+				item.creators.push(ZU.cleanAuthor(author, "author"))
+			}
+		}
+		item.publicationTitle = "El País";
+		item.ISSN = "0213-4608";
+		item.place = "Madrid";
```
[the remark this answers is not in the document]
Comment (adam3smith) [reply El Pais.js]: To the extent that citation styles want that information, El Pais (Madrid) is going to distinguish it from other El Pais papers which may exists e.g. in Latin Americal. ZDB also cites it as Madrid, so I'll leave this in.

File: El Pais.js [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+				"itemType": "newspaperArticle",
+				"title": "Hallada vacía la botella de vodka valorada en 1,1 millones de euros robada en Copenhague",
+				"creators": [],
+				"date": "2018-01-06",
+				"ISSN": "0213-4608",
+				"abstractNote": "Un obrero encontró el recipiente, hecho de oro y plata, y está aparentemente intacto y en poder de la policía",
+				"libraryCatalog": "elpais.com",
+				"place": "Madrid",
+				"publicationTitle": "El País",
+				"section": "Mundo_global",
```
[the remark this answers is not in the document]
Comment (adam3smith) [reply El Pais.js]: done -- I applied the title casing after replacing to get "Mundo Global" etc. in line with what the sections are called in the paper.

**Comment by adam3smith** [comment]
Thanks -- please have a look at updated version

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**